pygeoapi/provider/mvt_elastic.py

```diff
@@ -162,11 +164,20 @@ def get_tiles(self, layer=None, tileset=None,
             else:
                 url_query = ''
 
-            with requests.Session() as session:
-                session.get(base_url)
-                resp = session.get(f'{base_url}/{layer}/{z}/{y}/{x}{url_query}')  # noqa
-                resp.raise_for_status()
-                return resp.content
+            try:
+                with requests.Session() as session:
+                    session.get(base_url)
+                    resp = session.get(f'{base_url}/{layer}/{z}/{y}/{x}{url_query}')  # noqa
+                    resp.raise_for_status()
+                    return resp.content
+            # Client is sending an invalid request
+            except requests.exceptions.HTTPError as e:
```

Comment:
By the way this could also have other causes such as an internal server error at the server side. To be more specific about client side errrors, we could check if `400 <= e.response.status_code < 500`